modules/core/include/opencv2/core/base.hpp

```diff
@@ -263,6 +263,14 @@ enum BorderTypes {
     BORDER_ISOLATED    = 16 //!< do not look outside of ROI
 };
 
+enum PeronaMalikFluxFunctions
+{
+    CV_PERONA_MALIK_EXPONENTIAL = 0,
+    CV_PERONA_MALIK_INVERSE_QUADRATIC = 1
```

Comment:
Please avoid using of `CV_` prefix for enums (it is for legacy C-API only). These values are already in `cv::` namespace.
Please place this enum near `PeronaMalik` function (in the same module, no need to place in in core module)